src/site/content/en/learn/css/the-cascade/index.md

```diff
@@ -93,7 +93,7 @@ This means it gets more specificity than the <code>&lt;style&gt;</code> element<
 An inline `style` attribute with CSS declared in it will override all other CSS,
 regardless of its position, unless a declaration has `!important` defined.
 
-Position also applies in the order of your in a CSS rule.
+Position also applies to the order *inside* a CSS rule.
```

Comment:
```suggestion
Position also applies in the order of your CSS rule.
```
still clarifies, fixes the grammar, and introduces less change?

Reply:
agreed

Reply:
You want we to squash these commits?